RELEASE.md

```diff
@@ -0,0 +1,31 @@
+## Major / Minor Release
+
+- create a branch for the release e.g. `7.6.x`
+- provide a [PR](https://github.com/eclipse/che-machine-exec/pull/66) with bumping the [VERSION](https://github.com/eclipse/che-machine-exec/blob/master/VERSION) file to the `7.6.x` branch
+- [![Release Build Status](https://ci.centos.org/buildStatus/icon?subject=release&job=devtools-che-machine-exec-release/)](https://ci.centos.org/job/devtools-che-machine-exec-release/) CI is triggered based on the changes in the [release](https://github.com/eclipse/che-machine-exec/tree/release) branch (not `7.6.x`).
+
+In order to trigger the CI once the [PR](https://github.com/eclipse/che-machine-exec/pull/66) is merged to the `7.6.x` one needs to:
+
+```
+ git fetch origin 7.6.x:7.6.x
+ git checkout 7.6.x
+ git branch release -f 
+ git push origin release -f
+```
+
+CI will build an image from the [`release`](https://github.com/eclipse/che-machine-exec/tree/release) branch and push it to [quay.io](https://quay.io/organization/eclipse) e.g [quay.io/eclipse/che-machine-exec:7.6.0](https://quay.io/repository/eclipse/che-machine-exec?tab=tags&tag=7.6.0)
+
+The last thing is the tag `7.6.0` creation from the `7.6.x` branch
+
+```
+git checkout 7.6.x
+git tag 7.6.0
+git push origin 7.6.0
+```
+
```

Comment:
As asked above ... can this be done by a bot?

Reply:
well, yes sure - but this is what is expected to be done by the Productization team. The CI process can be changed, updated or fully reworked if needed

Reply:
Great! Just wondering if there were **technical reasons** for not yet doing this via bot, vs. time constraints ("we haven't done it yet, but we want it to be done") :)

Reply:
@benoitf could you please comment on that? I might be wrong, but I recall that in the past Eclipse foundation prohibited commits from bots